Source/DafnyDriver/TextLogger.cs

```diff
@@ -18,11 +18,11 @@ public class TextLogger {
     tw = parameters.TryGetValue("LogFileName", out string filename) ? new StreamWriter(filename) : outWriter;
   }
 
-  public void LogResults(List<(Implementation, VerificationResult)> verificationResults) {
-    var orderedResults = verificationResults.OrderBy(vr => vr.Item1.VerboseName);
-    foreach (var (implementation, result) in orderedResults) {
+  public void LogResults(IEnumerable<(string, VerificationResult)> verificationResults) {
```

Comment:
Could you add names to the elements of this tuple? Like `(string batchDescription, VerificationResult result)`